packages/web-components/fast-foundation/src/radio-group/radio-group.ts

```diff
@@ -95,7 +95,7 @@ export class RadioGroup extends FASTElement {
             }
         });
 
-        if (this.value === undefined) {
+        if (this.value === undefined && radioButtons && radioButtons.length > 0) {
```

Comment:
since radioButtons will always be at least an empty array, I think you can skip the radioButtons check and just look at the length

Reply:
done and done